addons/l10n_gh/__manifest__.py

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+{
+    'name': 'Ghana - Accounting',
+    'category': 'Localization/Account Charts',
+    'description': """
+This is the base module to manage the accounting chart for Ghana.
+===========================================================================
+
+* Chart of Accounts.
+
+    """,
+    'author': 'erpSOFTapp',
+    'license': 'AGPL-3',
+    'depends': [
+        'account',
+        'account_accountant',
```

Comment:
just account will do

Reply:
Yes WHT is withdrawn from the total amount on the invoice for these 2 countries.
When you say have it with tax report and P&L balance sheet are you referring to a separate xx_report module to incorporate these..can you clarify 

We will remove account_accountant

Reply:
No need to specify base below as well.  (as account depends on base)  

The tax report can already be defined in this module.  In the repartition_line_ids, you can add the plus/minus_tag_ids to refer to the tax report line that needs that base/tax amount added/subtracted.  At least for this VAT return the advantage is that we check if the necessary taxes are defined, ...  I see e.g. no fiscal positions and taxes for export. 

For the P&L and balance sheet, that would be through a l10n_gh_reports in enterprise.